site/download.md

```diff
@@ -65,14 +54,18 @@ Client guide | API docs
 [The Pulsar C++ client](../docs/latest/clients/Cpp) | [Doxygen docs](../api/cpp)
 
 
+<!--
 {% if site.archived_releases %}
 
-### Other releases
 
-| Release   | Download | Release notes                                                                                      |
-|:-------|:--------------------------------------------|--------------------------------------------|
+### Older releases
+
+Release | Download | Crypto files | Release notes
+:-------|:---------|:-------------|:-------------
 {% for version in site.archived_releases
-%} {{version}} | [http://archive.apache.org/dist/incubator/pulsar/pulsar-{{version}}](http://archive.apache.org/dist/incubator/pulsar/pulsar-{{version}}) | [Release notes v{{version}}](https://github.com/apache/incubator-pulsar/releases/tag/v{{ version }})|
+%} {{version}} binary | [pulsar-{{version}}-bin.tar.gz](http://www.apache.org/dist/incubator/pulsar/{{version}}/pulsar-{{version}}-bin.tar.gz) | [asc](http://www.apache.org/dist/pulsar/{{version}}/pulsar-{{version}}-bin.tar.gz.asc), [md5](http://www.apache.org/dist/pulsar/{{version}}/pulsar-{{version}}-bin.tar.gz.md5), [sha512](http://www.apache.org/dist/pulsar/{{version}}/pulsar-{{version}}-bin.tar.gz.sha512) | [Release notes v{{version}}](https://github.com/apache/incubator-pulsar/releases/tag/v{{ version }})
```

Comment:
Can we extract the base URL `http://www.apache.org/dist/incubator/pulsar/` into a variable to make the whole text smaller?

Reply:
I'm looking at the Jekyll documentation and as far as I know this can't be done inside of a `for` loop. But I'll let you know if I figure out a way to do that.